src/functions/revealLeft.tsx

```diff
@@ -0,0 +1,17 @@
+const revealLeft = () => {
+  let reveals = document.querySelectorAll('.reveal-left');
```

Comment:
use const for all variables (except i) as there are no value changes done afterward